.gitignore

```diff
@@ -234,6 +234,7 @@ FakesAssemblies/
 *.crt
 /src/api/AppointmentsDataLoader
 /src/api/Nhs.Appointments.Api/jon.settings.json
```

Comment:
src/api/Nhs.Appointments.Api/jon.settings.json
/src/api/Nhs.Appointments.Api/Properties/launchSettings.jon.json
Would these be better to be ignored in your git exclude file as they are you personal settings?